client/package-lock.json

```diff
@@ -24,7 +24,9 @@
         "react-burger-menu": "^3.0.9",
         "react-dom": "^18.3.1",
         "react-redux": "^9.1.0",
-        "react-router-dom": "^6.26.1"
+        "react-router-dom": "^6.26.1",
+        "react-toastify": "^10.0.5",
+        "redux-persist": "^6.0.0"
```

Comment:
as an idea, you using strapi, does it allow to store your cart items in the database? is it possible? instead of storing them locally with redux-persist. just consider it for future research and improvements.